tools/CMakeLists.txt

```diff
@@ -159,12 +159,17 @@ set(MANIFEST_COMMAND
     -o ${CMAKE_BINARY_DIR}/generated
     ${PARSE_MANIFEST_QUIET_FLAG})
 
+set(NO_BUILD_CMD_FOR_MANIFEST 1)
```

Comment:
If I understand correctly, the intention here is to set/unset this variable in the TF-M build script whenever we do not want to run the manifest command. Can you point me to how this variable will be used in the build script?